scripts/test.sh

```diff
@@ -0,0 +1,14 @@
+#! /bin/bash
+set -ex
+
+SCRIPT_DIR=$(cd $(dirname "$0") && pwd -P)
+ROOT=$(dirname $SCRIPT_DIR)
+
+if [ "$1" = "--install" ]; then
+    cargo install cddl
+    npm install parse5
+fi
+$ROOT/scripts/cddl/generate.js
```

Comment:
Mind adding the comment for `Extract CDDL content from spec into files` which was previously in `test.yml`?